R-packages/evalcast/R/plot_calibration.R

```diff
@@ -48,19 +50,16 @@ plot_calibration <- function(scorecard,
       geom_line(color = "red") +
       geom_point(color = "red") +
       geom_abline(slope = 1, intercept = 0) +
-      labs(x = "Nominal quantile level",
+      labs(x = "Quantile level",
            y = "Proportion",
-           title = sprintf(
-             "%s (ahead %s) - Calibration",
-             name,
-             ahead))
+           title = sprintf("%s (ahead %s): Calibration", name, ahead))
   }
   g +
     facet_wrap(~ forecast_date) +
     geom_vline(xintercept = c(alpha, 1 - alpha), lty = 2) +
     xlim(0, 1) +
     ylim(0, 1) +
-    theme_bw()
+    theme_bw() + theme(legend.position = legend.position)
```

Comment:
should add theme to line with @importFrom ggplot2 so that R CMD check doesn't complain

Reply:
Ok sure.